makelib/helmoci.mk

```diff
@@ -0,0 +1,108 @@
+# Copyright 2023 The Upbound Authors. All rights reserved.
```

Comment:
This is the second or 3rd place I've seen this copied around to recently. Would it make sense to move this the build submodule?

Reply:
we deprecated the submodule - so i wonder what is the way to go

Reply:
I thought we deprecated the build submodule, but we have https://github.com/crossplane/build for OSS repos 🤷‍♂️

Reply:
okay will open a PR in crossplane/build and will followup